lyrebird/src/window.rs

```diff
@@ -174,9 +174,9 @@ impl MainWindow
 				// tab and ask it what it thinks of this
 				let operation = match self.activeTab
 				{
-					Tab::LibraryTree => self.libraryTree.handleKeyEvent(key),
-					Tab::Options => self.optionsPanel.handleKeyEvent(key),
-					Tab::Playlists => self.playlists.handleKeyEvent(key),
+					Tab::LibraryTree => self.libraryTree.handleKeyEvent(*key),
+					Tab::Options => self.optionsPanel.handleKeyEvent(*key),
+					Tab::Playlists => self.playlists.handleKeyEvent(*key),
```

Comment:
Given this, does it make sense to alter handleKeyEvent() in those types too to take a `&KeyEvent`?

Reply:
I think that would make sense.